subprojects/core/src/main/java/org/gradle/api/internal/changedetection/state/GradleUserHomeScopeFileTimeStampInspector.java

```diff
@@ -51,17 +52,20 @@ public void attach(CachingFileHasher fileHasher) {
     public void afterStart() {
         updateOnStartBuild();
         currentTimestamp = currentTimestamp();
+        isBaseTimestampHighPrecision = isHighTimestampPrecision(currentTimestamp);
```

Comment:
```suggestion
        currentTimestampHighPrecision = isHighTimestampPrecision(currentTimestamp);
```

We use `currentTimestamp` as the variable name.